### PR TITLE
[MTS/TC] [70405604] Add change handler

### PR DESCRIPTION
**Pull request by lfender6445**
Pivotal Story 1: https://www.pivotaltracker.com/story/show/70405604
Pivotal Story 2: https://www.pivotaltracker.com/story/show/76127754

We're introducing some functionality that will allow us to tag change events on `<select>` elements. We extracted some functionality from the base object in this library. Now, there are two CoffeeScript classes; one deals with clicks, and the other deals with change events. Using RequireJS in this library to include those new CoffeeScript files means we will have to edit the RequireJS configuration files of our applications. We'll have to  include jquery-autotagging in the `packages` section instead of the `paths` section.
- [Merge Me First](https://github.com/primedia/ag/pull/1663)
- We MUST ensure we update `bower.json` and the RequireJS configuration files for:
  - ag
  - ag_sites
  - m.apartmentguide
  - m.newhomeguide
  - ag_mobile_api
  - wp_blog_themes
  - rent

All sites that use jquery-autotagging need to be updated. Rentals uses webtrends so we can ignore it.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "jquery-autotagging",
-  "version": "1.0.15",
+  "version": "1.0.16",
```
Comment (michaelstalker) [inline bower.json]: @tommyc: @pasha had suggested bumping the major version. We need to change the RequireJS configuration files in each app that uses this, so our code counts as a breaking change. What do you think?

**Comment by michaelstalker** [comment]
@lfender6445 says: @tommyc @mstalker - don't forget to push a tag post merge. i've already bumped version in bower.json

File: bower.json (hunk 1 above)
Comment (lfender6445) [inline bower.json]: Yes sounds like a plan to me!

Comment (michaelstalker) [inline bower.json]: @lfender6445 I bumped the version to 2.0.0 in `bower.json` and `package.json`.

**Comment by pasha** [comment]
:+1:

**Pull request merged**